### ISSUE TITLE
More language things

### ISSUE THREAD

**Issue by ninclay** [emilienschultz/activetigger#238]
Test
No test data set have been created. Do you want to upload your own test set?
-> No test data set **has** been created.

Export
Export train data
-> Export training data

**Comment by emilienschultz** [comment]
Done
